src/moduleResolver/ModuleResolver.ts

```diff
@@ -80,22 +80,32 @@ function initModule(props: { absPath: string; bundleContext: IBundleContext; ctx
     if (module.isExecutable) {
       module.parse();
       const transformerResult = module.transpile();
+
       for (const item of transformerResult.requireStatementCollection) {
-        const source = item.statement.arguments[0].value;
-
-        const resolvedModule = resolve({
-          bundleContext: props.bundleContext,
-          ctx: props.ctx,
-          importType: item.importType,
-          parent: module,
-          statement: source,
-        });
-
-        module.moduleSourceRefs[source] = resolvedModule;
-        // re-writing the reference
-        item.statement.callee.name = BUNDLE_RUNTIME_NAMES.ARG_REQUIRE_FUNCTION;
-        item.statement.arguments[0].value = resolvedModule.id;
-        delete item.statement.arguments[0].raw;
+        // no value.. import() or require();
+        if (!item.statement.arguments[0]) continue;
+        // regular import checks
+        if (item.statement.arguments.length === 1) {
+          const source = item.statement.arguments[0].value;
+          // we're importing some weird stuff... require(1) ie.
+          if (typeof source !== 'string') continue;
+          const resolvedModule = resolve({
+            bundleContext,
+            ctx,
+            importType: item.importType,
+            parent: module,
+            statement: source,
+          });
+          if (!resolvedModule) continue;
```

Comment:
Let's make sure that if we're not able to resolve that module we do:

* What I mentioned above
*  Also make sure that logging is in place